cg/server/invoices/views.py

```diff
@@ -127,6 +133,7 @@ def invoice_template(invoice_id):
 
 @BLUEPRINT.route('/<int:invoice_id>/invoice_file/<cost_center>')
 def modified_invoice(invoice_id, cost_center):
+    return redirect(url_for('admin.index'))
```

Comment:
This is just a quickfix?

Reply:
It is, I felt it took to long to figure out how to fix it. I guess we should plan implementing a solution in a coming sprint or ask Maya to take care of it since she created those views.

Reply:
@mayabrandi ping pong. You should prolly know that we disabled the invoices on clinical-api for now. We don't seem to be able to get them safely behind admin views.